workflow/src/main/java/com/synopsys/integration/alert/workflow/scheduled/frequency/ProcessingTask.java

```diff
@@ -72,7 +72,9 @@ public DateRange getDateRange() {
     public void runTask() {
         DateRange dateRange = getDateRange();
         List<AlertNotificationModel> notificationList = read(dateRange);
+        logger.info("Processing {} notifications.", notificationList.size());
         List<DistributionEvent> distributionEvents = notificationProcessor.processNotifications(getDigestType(), notificationList);
+        logger.info("Sending {} events for notifications.", distributionEvents.size());
```

Comment:
Should one of these be debug as well? We don't want to spam too much

Reply:
Same as above.